app/src/main/java/com/opennotes/settings/data/repository/DataStoreRepository.kt

```
@@ -97,33 +106,37 @@ class DataStoreRepository
-                            try {
-                                ThemeMode.valueOf(themeModeString)
-                            } catch (_: IllegalArgumentException) {
-                                // If migration needed from legacy settings
-                                migrateLegacyThemeMode(prefs, defaultSettings)
-                            }
-                        } else {
-                            // Migration from legacy settings
-                            migrateLegacyThemeMode(prefs, defaultSettings)
+                        try {
+                            ThemeMode.valueOf(themeModeName)
+                        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)